### DISCUSSION TITLE
Log looking for plot that doesn't exist

### DISCUSSION THREAD

**Discussion by moechenche** [ericaltendorf/plotman#541] [Q&A]
...starting plot loop
Traceback (most recent call last):
  File "/home/moechenche/.local/lib/python3.8/site-packages/plotman/manager.py", line 135, in maybe_start_new_plot
    open_log_file = open(logfile, 'x')
FileNotFoundError: [Errno 2] No such file or directory: '/home/chialogs/2021-05-17T20_35_28.875256-05_00.log'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/moechenche/.local/bin/plotman", line 8, in <module>
    sys.exit(main())
  File "/home/moechenche/.local/lib/python3.8/site-packages/plotman/plotman.py", line 145, in main
    wait_reason = manager.maybe_start_new_plot(cfg.directories, cfg.scheduling, cfg.plotting)
  File "/home/moechenche/.local/lib/python3.8/site-packages/plotman/manager.py", line 153, in maybe_start_new_plot
    raise Exception(message) from e
Exception: Unable to open log file.  Verify that the directory exists and has proper write permissions: '/home/chialogs/2021-05-17T20_35_28.875256-05_00.log'

That Plot doesn't exist, how do i get out of this issue?
Here is what is in the folder

moechenche@moechenche-Precision-Tower-5810-DBE:~$ ls -la /home/moechenche/chialogs
total 8
drwxrwxr-x  2 moechenche moechenche 4096 May 17 19:35 .
drwxr-xr-x 24 moechenche moechenche 4096 May 17 19:16 ..
-rw-rw-r--  1 moechenche moechenche    0 May 17 17:49 2021-05-17T17_49_05.266998-05_00.log
-rw-rw-r--  1 moechenche moechenche    0 May 17 17:49 2021-05-17T17_49_05.269952-05_00.log
-rw-rw-r--  1 moechenche moechenche    0 May 17 18:21 2021-05-17T18_21_29.864953-05_00.log
-rw-rw-r--  1 moechenche moechenche    0 May 17 18:21 2021-05-17T18_21_50.856215-05_00.log
-rw-rw-r--  1 moechenche moechenche    0 May 17 18:23 2021-05-17T18_23_36.332432-05_00.log
-rw-rw-r--  1 moechenche moechenche    0 May 17 18:25 2021-05-17T18_25_36.240827-05_00.log
-rw-rw-r--  1 moechenche moechenche    0 May 17 18:26 2021-05-17T18_26_33.326649-05_00.log
-rw-rw-r--  1 moechenche moechenche    0 May 17 18:30 2021-05-17T18_30_56.605162-05_00.log
-rw-rw-r--  1 moechenche moechenche    0 May 17 18:35 2021-05-17T18_35_17.822202-05_00.log
-rw-rw-r--  1 moechenche moechenche    0 May 17 18:39 2021-05-17T18_39_05.784321-05_00.log
-rw-rw-r--  1 moechenche moechenche    0 May 17 18:50 2021-05-17T18_50_23.220568-05_00.log
-rw-rw-r--  1 moechenche moechenche    0 May 17 18:54 2021-05-17T18_54_37.952263-05_00.log
-rw-rw-r--  1 moechenche moechenche    0 May 17 19:15 2021-05-17T19_15_56.239850-05_00.log
-rw-rw-r--  1 moechenche moechenche    0 May 17 19:16 2021-05-17T19_16_37.541103-05_00.log
-rw-rw-r--  1 moechenche moechenche    0 May 17 19:17 2021-05-17T19_17_00.880517-05_00.log
-rw-rw-r--  1 moechenche moechenche    0 May 17 19:35 2021-05-17T19_35_08.692489-05_00.log

**Comment by moechenche** [comment]
i started plot from the Chia Gui, now this is no longer an issue, not sure how that fixed it but it did, this error came back once plotman tried to start

**Reply by moechenche** [reply]
I still have this issue. plotman stopped with this error, how do I fix it?  the .log file it's looking for doesn't exist

**Comment by maxbelz** [comment]
same here , hdd have not-logged plots
